.github/workflows/linters.yml

```diff
@@ -0,0 +1,161 @@
+# yaml-language-server: $schema=https://json.schemastore.org/github-workflow
```

Comment:
Can not not enforce the double quotes in YAML, I dislike the visual clutter.

Reply:
This makes Viktor very sad.

Arrives in a minute.